ionoscloud/resource_nic_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build compute || all || nic
-
```

Comment:
@cristiGuranIonos shouldn't we add cases for all the scenarios that can occur related to changes/modifications? For example when you have a flowlog and you delete it, or when you modify a `force-new` field and so on.

Reply:
testAccCheckNicConfigUpdate - takes care of flowlog update
testAccCheckNicConfigUpdateIpv6 - takes care of deletion